common/scala/build.gradle

```diff
@@ -25,7 +25,7 @@ dependencies {
     compile 'commons-codec:commons-codec:1.9'
     compile 'commons-io:commons-io:2.4'
     compile 'commons-collections:commons-collections:3.2.2'
-    compile 'org.apache.kafka:kafka-clients:0.10.2.1'
+    compile 'org.apache.kafka:kafka-clients:0.11.0.1'
```

Comment:
Starting with kafka 0.10.2, they support both forward & backward version mismatch of the server and clients.  So we don't have to be running the 0.11 server to use the 0.11 client.

I do think upgrading the kafka server is a good idea to keep current, so I am planning on submitting a separate PR that just does that.  By isolating that change it will be easier to roll it back and/or performance test it separately.